tests/test_corpora.py

```diff
@@ -38,10 +38,26 @@ def build_index_corpus(directory):
     return quantgov.corpora.IndexDriver(str(index_path))
 
 
+def build_s3_corpus(directory):
+    rows = []
+    for letter, number, path in (
+            ('a', '1', 'quantgov_tests/first.txt'),
+            ('b', '2', 'quantgov_tests/second.txt')
+    ):
+        rows.append((letter, number, path))
+    index_path = directory.join('index.csv')
+    with index_path.open('w', encoding='utf-8') as outf:
+        outf.write(u'letter,number,path\n')
+        outf.write(u'\n'.join(','.join(row) for row in rows))
+    return quantgov.corpora.S3Driver(str(index_path),
+                                     bucket='quantgov-databanks')
```

Comment:
Will people outside the core dev team be able to run these tests?

Reply:
As long as they have aws credentials for boto I believe so, since the bucket is public.